ddtrace/tracer/option.go

```diff
@@ -141,6 +141,11 @@ func newConfig(opts ...StartOption) *config {
 		}
 	}
 	c.logStartup = boolEnv("DD_TRACE_STARTUP_LOGS", true)
+	if boolEnv("DD_TRACE_ANALYTICS_ENABLED", false) {
+		globalconfig.SetAnalyticsRate(1.0)
+	}
+	c.runtimeMetrics = boolEnv("DD_RUNTIME_METRICS_ENABLED", false)
```

Comment:
Not sure what you did with this `boolEnv` thing and why the other's exported :)

Reply:
Yes, I'll remove the tracer one and use the internal `BoolEnv`